src/demo/js/script.js

```diff
@@ -8,7 +8,8 @@ let preview = {
     vCenter: "false",
     multiline: "false",
     width: "400",
-    height: "50"
+    height: "50",
+    speed: "5"
```

Comment:
This should match the default in other places which is 10
```suggestion
    speed: "10"
```